python/Ganga/GPIDev/Lib/Job/Job.py

```diff
@@ -2091,20 +2089,19 @@ def __setattr__(self, attr, value):
 
             # reduce duplicate values here, leave only duplicates for LCG,
             # where we can have replicas
-            uniqueValuesDict = []
-            uniqueValues = []
+            uniqueValuesDict = oDict()
```

Comment:
In this case why does it need to be a `dict`? Would not a list suffice like the code used to have?